include/TrustWalletCore/TWCoinType.h

```diff
@@ -104,6 +104,7 @@ enum TWCoinType {
     TWCoinTypeBoba = 10000288,
     TWCoinTypeMetis = 1001088,
     TWCoinTypeAurora = 1323161554,
+    TWCoinTypeEvmos = 9001,
```

Comment:
For EVM-chains there is no real coinID, we derive it from ChainID, value should be `10009001`.